### ISSUE TITLE
Easy for absolute beginners to get confused on their first level

### ISSUE THREAD

**Issue by mogsie** [codecombat/codecombat#2306]
I witnessed my son's first lines of programming, and it was exhilerating.  And although it went beyond expectations (he coded the third to fifth level correctly first time without hitting "run"), I have a few gripes with the way the editor screen pops up.

He had never seen a line of code, not even _scratch_ or anything before, and a lot of things happen the first time you open the editor:

- The hero starts to move
- The progress bar starts to move
- Code in the editor highlights
- Yellow arrows pop up
- blue pop-ups (translated) pop up.
- a voice says to do this or that.

It all happens quite quickly, and it is a bit challenging (especially for non-English speakers) to keep up. Easy for a professional programmer to understand, but overwhelming for a child who has never seen code. In my opinion, I think it should be much calmer. Perhaps something more modal like this:

 - Nothing moves when you open the level, i.e. calmness.
 - The hero is highlighted, perhaps a spotlight effect used on the hero. No text is necessary.
 - A mouse click anywhere advances this "slide show". (The characteristic yellow arrow could be used to guide the user to click somewhere). The point is to ensure that the child sees the hero, recognises the hero they just chose in the previous screen.
 - The next thing highlighted is the editor window.  Also using that spotlight.  English-natives will easily read "self.moveLeft()". Perhaps the comments should say (localized) that "moveLeft means the hero moves left a bit" instead of the current list of goals.
 - Yet another click (anywhere on the screen) to confirm that the child has seen the code.
 - Then guide the child to hit the "run" button, (possibly using the yellow arrow), but now the modality is gone, allowing for exploratory behaviour.  I want to get the first-time users to hit that Run button, before writing any code.

I think this would introduce the link between the hero, the code, and the run button, so that the child understands what's happening without any guidance, e.g. if the child does this on her own or if the parent is unfamiliar with coding.

**Comment by popey456963** [comment]
These are very good suggestions, thanks!  We're hoping that non-English speakers will still be able to keepup because our localization is coming on [very well](https://codecombat.com/contribute/diplomat).  I do think that there are a few too many things happening on the first level, and I'll tag this as an enhancement.

**Comment by mogsie** [comment]
Yes, I think once you're past the first level, you've grokked it. After level 1, I think it's all pretty good up until "variables"-levels; they need some TLC too.  But a child that randomly hears about CC and ends up no level 1 will often not make it past the first level, simply because they don't understand the connection between the code and the movement of the character.

**Comment by txomon** [comment]
I have had a similar experience, non-english kid, too much movement in the screen.

I can't say I would do it that slow, but maybe a mini intro, in the interface itself, where the kid would see the connection between the hero and the code.

Also, they don't see the code as commands nor statements
